main.py

```diff
@@ -120,11 +119,7 @@ def _make_chart_layout(self, path: str) -> html.Div:
     @staticmethod
     def _redirect_to_chart_view(from_agent, to_agent: AgentID) -> dcc.Location:
         path = ChartView.encode_path(from_agent, to_agent)
-        return dcc.Location(pathname=path, id="")
-
-    @property
-    def mesh(self) -> MeshResults:
-        return self._cached_repo.get_mesh_test_results()
+        return dcc.Location(pathname=path, id=IndexView.URL, refresh=False)
```

Comment:
Where is the `path` coming from?

Reply:
From a line not visible here: path = ChartView.encode_path(from_agent, to_agent)